src/devcenter/azext_devcenter/aaz/latest/devcenter/admin/network_connection/_create.py

```diff
@@ -90,32 +89,32 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.domain_name = AAZStrArg(
             options=["--domain-name"],
             arg_group="Properties",
-            help="Active Directory domain name",
+            help="Active Directory domain name.",
         )
         _args_schema.domain_password = AAZStrArg(
             options=["--domain-password"],
             arg_group="Properties",
-            help="The password for the account used to join domain",
+            help="The password for the account used to join domain.",
```

Comment:
We support prompt-input for these password argument. You can reference these two PRs https://github.com/Azure/aaz-dev-tools/pull/238 and https://github.com/Azure/azure-cli/pull/26134

Reply:
Fixed